src/katsdpcontroller/product_controller.py

```diff
@@ -1498,6 +1498,15 @@ async def beam_delays(self, stream_name: str, coefficient_sets: Sequence[str]) -
             timeout=DELAYS_TIMEOUT,
         )
 
+    async def vlbi_delay(self, stream_name: str, delay: float) -> None:
+        # TODO: Check against self.state for CBF case
+        # - ProductState.IDLE basically means running. To determine "capturing"
+        #   we need to use the `transmit_state` for the graph node coresponding
+        #   to the multicast output.
```

Comment:
For now put in the same logic as other requests i.e.
```python
        if self.state not in {ProductState.CAPTURING, ProductState.IDLE}:
            raise FailReply(f"Cannot set VLBI delay in state {self.state}")
```

But leave a TODO to also check the `transmit_state`; that's extra for this request compared to others.